oca_dependencies.txt

```diff
@@ -1,15 +1,3 @@
 # List the OCA project dependencies, one per line
 # Add a repository url and branch if you need a forked version
-#
-# Examples
-# ========
-#
-# To depend on the standard version of sale-workflow, use:
-# sale-workflow
-#
-# To explicitely give the URL of a fork, and still use the version specified in
-# .travis.yml, use:
-# sale-workflow https://github.com/OCA/sale-workflow
-#
-# To provide both the URL and a branch, use:
-# sale-workflow https://github.com/OCA/sale-workflow branchname
+project_categ https://github.com/OCA/project 8.0
```

Comment:
You just need name of the dependency repo.
A line with just `project` is enough.